README.md

```diff
@@ -265,7 +291,7 @@ Connection response are provided via callbacks.
 ```javascript
 user.reconnect('ACCOUNT_ID', {
   // additional options
-}).onConnection(account => {
+}).onConnection((account, message) => {
```

Comment:
Same here, not sure if `message` is still relevant

Reply:
Discussed in thread, reverted.